compass/core/_scrapers/member.py

```diff
@@ -288,7 +288,7 @@ def get_roles_tab(
             status_with_review = cells[5].text_content().strip()
             if status_with_review.startswith("Full Review Due "):
```

Comment:
Has this been tested? I don't think this will fix the issue as this condition will still fail...

Reply:
This is why we do not try to fix things at midnight. Also see #6! Meta-fix incoming...